Ch04/numbercrunch.cpp

```diff
@@ -0,0 +1,28 @@
+#include "../std_lib_facilities.h"
+
+double sum(const vector<double>& v) {
+    double sumofnum = 0.0; 
+    for (double i:v) {
+        sumofnum += i;
+    };
+    return sumofnum;
+}
+
+double median(const vector<double>& v) {
+	int check = v.size()%2;
```

Comment:
This is clear enough to be put straight in the `if` statement without the need for variable `check`.